### DISCUSSION TITLE
Feature Request - Select Editor per Usergroup

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#14438] [4. Rejected Features]
For now you can select a default-Editor per User or a default Editor for all Users. But what if the registered Users should not have an Editor but the Administrators? Then you would need to set up the default Editor to none and assign tinyMCE to each Admin. I think it could be nice to have an Default Editor Setting in the Group Params for each Group. 
Any thoughts?

**Comment by mbabker** [comment]
> But what if the registered Users should not have an Editor but the Administrators

This would mean you're giving users a plain text area and basically forcing them to write in HTML versus having a WYSIWYG option available to them.  At that point, I'd question why you're not just using a `<textarea>` form element or hardcoding the form to only use the none editor if you really need this type of configuration.

I don't think this should be changed on a per-group basis.  Also remember here that a user > group > global hierarchy for any parameter introduces a lot of problems in general.  When a user is assigned to multiple groups, what should the parameter value be?  If you say "most privileged", how do we know what that means in a given context?

**Comment by brianteeman** [comment]
Don't forget that in 3.7 you can completely customise tinymce per usergroup. That should cover most real world examples

**Comment by coolcat-creations** [comment]
@brianteeman yes, but i can´t remove the buttons there :( I would need to set ACL for each button seperately... I ran into this because i want to create actually really a textarea instead of editor in the frontend. But it´s still not possible to make an alternative Layout for editing contents. It´s possible to create an alternative Layout with a Menütype, but at article edit the edit button can´t be changed in core to take a specific layout. With plugins of course but not with core.

Specific Usecase: I provide only Title, Alias and the Custom Fields for the User to submit content. All other Joomla Fields are removed. As said - this works only for submitting Menutype but not for edit.

**Comment by coolcat-creations** [comment]
An alternative Layout for EDITING an Article inside a specific Category would solve this issue too. It´s just like the plugin from @dgt41 but seeing something like that in core would be great.

**Comment by brianteeman** [comment]
iirc The buttons are not the editor they afre plugins that are being displayed in the editor

**Comment by dgrammatiko** [comment]
@brianteeman true, but specifically for tinymce we render those button inside the editor, in other words they are part of the tinymce configuration parameters

**Comment by SharkyKZ** [comment]
For what it's worth, this is super simple to do with a plugin.

**Comment by rdeutz** [comment]
The Joomla Maintainer Team decided on the 25.02.2026 meeting to reject this feature request because it will cause more problems for too less positive outcome
